src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToSubmitHandler.java

```diff
@@ -51,17 +56,31 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
 
         PreSubmitCallbackResponse<SscsCaseData> callbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
 
-        State state = callback.getCaseDetails().getState();
-        HearingRoute hearingRoute = sscsCaseData.getSchedulingAndListingFields().getHearingRoute();
+        ReserveTo callbackReserveTo = callbackResponse.getData().getSchedulingAndListingFields().getReserveTo();
+        SchedulingAndListingFields caseDataSnlFields = sscsCaseData.getSchedulingAndListingFields();
+
+        if (nonNull(callbackReserveTo)) {
+            YesNo callbackReservedDtj = callbackReserveTo.getReservedDistrictTribunalJudge();
+            ReserveTo caseDataReserveTo = caseDataSnlFields.getReserveTo();

```

Comment:
Any need to null check this or do we always expect a value? Same could be said for above (even if main object is not null do we expect null fields). Is it worth coding around this or not?

Reply:
You are right, the ReserveTo does come through the handler initilised with null fields if none are selected.
I will therefore remove this null check and related test.
![image](https://user-images.githubusercontent.com/110455092/219637020-fbfde17d-b351-4bbf-8c18-9ea77ceeb103.png)

Reply:
We have decided it is ok to keep the null check and all ReserTo-related logic contained in the block. It is not intrusive and prevents regression failures.